recipes/blaze/all/conandata.yml

```diff
@@ -0,0 +1,25 @@
+sources:
```

Comment:
Do you need all those version? Maybe you could provide only 3.7

Reply:
No, I don't _need_ all those versions. A long time ago I provided a recipe for Blaze 3.4 and assumed that  due to backwards compatibility it would be reasonable to provide every release since then. If you feel that this is not necessary and if this would only incur an unnecessary load on your build systems, I gladly remove all releases except Blaze 3.7. In that case, please also advise on how I should handle updates in the future: Should I add new releases or replace an old release with the new release?

Reply:
@igl42 unless it is strictly required and there are no breaking changes between versions, please add just the latest version